src/main/java/org/mitre/synthea/export/BB2RIFExporter.java

```diff
@@ -798,13 +864,19 @@ private void exportPrescription(Person person, long stopTime) throws IOException
   @Override
   public void flush() throws IOException {
     beneficiary.flush();
-    //    beneficiaryHistory.flush();
+    beneficiaryHistory.flush();
     inpatient.flush();
-    //    outpatient.flush();
-    //    carrier.flush();
-    //    prescription.flush();
+    outpatient.flush();
+    carrier.flush();
+    prescription.flush();
   }
 
+
+  public interface GetEnumValueInterface {
+    public int getEnumValue( String field );
+  }
```

Comment:
What is this? It doesn't seem to be used anywhere that I can see.

Reply:
nice catch, I'll delete it